src/view/sort-view.js

```diff
@@ -1,10 +1,23 @@
 import AbstractView from '../framework/view/abstract-view';
 
 export default class SortView extends AbstractView {
+  #handleSortButtonClick = null;
+  constructor({ onSortButtonClick }) {
+    super();
+
+    if (!onSortButtonClick) {
+      throw new Error('Parameter "onSortButtonClick" doesn\'t exist');
+    }
+
+    this.#handleSortButtonClick = onSortButtonClick;
+
+    this.element.addEventListener('click', this.#onSortButtonClick);
```

Comment:
Т.к. у нас есть `<form>`, мы можем использовать эвент `change` и тогда будет происходить только 1 событие и не придётся фильтровать клик с лейбла

```suggestion
    this.element.addEventListener('change', this.#onSortButtonClick);
```